src/components/Calendar/index.js

```diff
@@ -77,6 +78,18 @@ class Calendar extends PureComponent {
   }
   focusToDate = (date, props = this.props, preventUnnecessary = true) => {
     if (!props.scroll.enabled) {
+      if (preventUnnecessary && props.preventUnnecessaryRefocus) {
+        const focusedDateDiff = differenceInCalendarMonths(
+          date,
+          this.state.focusedDate,
+          this.dateOptions
+        );
```

Comment:
`differenceInCalendarMonths` seems to take only two arguments and options is redundant, can you remove that? :) 
(dateLeft: Date | number, dateRight: Date | number)